src/cappa/help.py

```diff
@@ -17,12 +17,12 @@
 from cappa.output import Displayable
 from cappa.subcommand import Subcommand
 from cappa.type_view import Empty
-from cappa.typing import assert_type
+from cappa.typing import T, assert_type
```

Comment:
I'm not a fan of this importing `T`, I think each `TypeVar` should be defined for its own context. But I decided to import it here because this pattern is used elsewhere.